api/package.json

```diff
@@ -74,14 +74,16 @@
 		"README.md",
 		"example.env"
 	],
+	"peerDependencies": {
+		"@directus/extensions-sdk": "^9.0.0"
```

Comment:
We should probably pin this to the the latest version similar to the other `@directus/*` dependencies to prevent issues with using multiple versions of `@directus/shared`.

Reply:
Yeah, I think that makes sense 👍